packages/react/react/src/hooks/useVisibilityEvent.spec.tsx

```diff
@@ -0,0 +1,41 @@
+import { fireEvent, render, renderHook } from '@testing-library/react';
+import { useState } from 'react';
+import { useVisibilityEvent } from './useVisibilityEvent';
+
+type VisibilityState = Document['visibilityState'];
+
+describe('useVisibilityEvent', () => {
+  it('When a user leaves the page.', () => {
+    const callbackMock = jest.fn();
+    renderHook(() => useVisibilityEvent(callbackMock));
+    expect(document.visibilityState).toBe('visible');
+
+    Object.defineProperty(document, 'visibilityState', { value: 'hidden' });
+    fireEvent(document, new Event('visibilitychange'));
+
+    expect(document.visibilityState).toBe('hidden');
+  });
+});
+
+describe('useVisibilityEvent for component', () => {
+  it('When a user leaves the page (component).', () => {
+    function TestComponent() {
+      const [visitable, setVisitable] = useState<VisibilityState>('visible');
+      const visibleCallbackFunc = (visibilit: VisibilityState) => {
+        setVisitable(visibilit);
+      };
+      useVisibilityEvent(visibleCallbackFunc);
+      return <div>{visitable}</div>;
+    }
+    const { getByText } = render(<TestComponent />);
```

Comment:
Note that in the latest testing library we use `screen.getByText()` instead of the getByText returned by render. Why not follow the latest convention?

https://testing-library.com/docs/react-testing-library/example-intro